test/CustomAbstracts.test.js

```diff
@@ -28,6 +28,7 @@ test(`Custom Abstracts: add a custom abstract`, t => {
     _addItem(editor, 'custom-abstract')
   }, 'adding new custom abstract should not throw error')
   t.equal(editor.findAll(customAbstractSelector).length, 1, 'there should be one custom abstract')
+  ensureValidJATS(t, app)
```

Comment:
@Integral I'd like you take a look of a new helper I have added in the last PR. Let's add this wherever we think it makes sense, i.e. whenever we have created a bunch of content, this checker makes sure that our exporter is creating valid JATS. Of course this is a very implicit way of testing the exporter, but it is kind of easy to include those checks every once in a while.

Reply:
Great, thanks!